admin/models/completed-checks.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const mongoose = require('mongoose')
+mongoose.Promise = global.Promise
+const Schema = mongoose.Schema
+
+const CompletedChecks = new Schema({
+  data: {
+    type: Object,
```

Comment:
@GeorgeChatzigiannis @GuyHarwood - do we want the model to be more restrictive?  Checking that all required keys are present?

Reply:
@jon-shipley we discussed this yesterday.  In hindsight, i think we can be fairly confident about the keys that are going to be used.  But if we do make it more restrictive we have to be mindful that this will need changing, should we introduce new keys.

Reply:
fine